gwcs/models.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, unicode_literals, print_function
+
+import numpy as np
+
+from astropy.modeling.core import Model
+from astropy.modeling.parameters import Parameter
+
+try:
+    from scipy.interpolate import interpn
```

Comment:
Also as I discovered when I first tried to run this, `interpn` is only available as of scipy 0.14 (my windows machine is still on 0.13, though clearly I need to upgrade!) 

Maybe this should have an explicit version check using `astropy.utils.minversion`.  It could also fall back on using `interp1d` and `interp2d` explicitly for the 1D and 2D cases...